package.json

```diff
@@ -18,11 +20,15 @@
     "axios": "^0.19.2",
     "bcrypt": "^5.0.1",
     "express": "^4.17.1",
+    "jsonwebtoken": "^8.5.1",
     "morgan": "^1.10.0",
     "pg": "^8.2.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-router-dom": "^5.2.0",
+    "react-router-dom": "^5.3.0",
+    "react": "^16.14.0",
```

Comment:
react, react-router-dom, and react-dom are repeated

Reply:
complete